presto-common/src/main/java/com/facebook/presto/common/RuntimeMetric.java

```diff
@@ -30,6 +31,7 @@
 public class RuntimeMetric
 {
     private final String name;
```

Comment:
Do we still need the `name` here since it's duplicate information?

Reply:
I'm kinda cautious to remove the existing field from the protocol. Some components might rely on it.

Reply:
Looks like the [Velox](https://github.com/facebookincubator/velox/blob/fdbbfd2354075da80e98f18055d744172f15e15f/velox/common/base/RuntimeMetrics.h#L25) definition doesn't have it, so to make it more consistent, can we at least check what components are relying on it and would it be easy to remove?

Reply:
I'm afraid this is another breaking change. The main reason we choose to keep the map structure instead of converting into array is that we cannot remove the existing information.